src/main/java/ch/wisv/areafiftylan/products/model/order/Order.java

```diff
@@ -93,6 +93,11 @@ public float getAmount() {
         return price;
     }
 
+    @JsonView(View.OrderOverview.class)
+    public float getAmountTickets() {
```

Comment:
Why is this a `float`? `Integer` or `int` seems more suitable

Reply:
Fixed